libs/generators/src/lib/generators/generators.component.scss

```diff
@@ -0,0 +1,42 @@
+:host {
+  display: flex;
+  flex-direction: column;
+  flex-grow: 1;
+  height: calc(100vh - 65px);
+  
+    .container {
```

Comment:
grid is not needed here, the nav should take it's size and the "content" should get all other size